DevEdu.API/DevEdu.Business.Tests/Data/CourseData.cs

```diff
@@ -0,0 +1,87 @@
+﻿using DevEdu.DAL.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DevEdu.Business.Tests
+{
+    public class CourseData
+    {
+        public CourseData()
+        {
+
+        }
+        /// <summary> This method returns CourseTopicDto, you can choose 1-3 version.</summary>
+        public static CourseTopicDto GetCourseTopicDto(int vesionOfDto)
+        {
+            CourseTopicDto courseTopicDto;
+            switch (vesionOfDto)
+            {
+                case 1:
+                    courseTopicDto = new CourseTopicDto { Position = 3 };
+                    break;
+                case 2:
+                    courseTopicDto = new CourseTopicDto { Position = 5 };
+                    break;
+                case 3:
+                    courseTopicDto = new CourseTopicDto { Position = 11 };
+                    break;
+                default:
+                    courseTopicDto = new CourseTopicDto { Position = 1 };
+                    break;
+            }
+
+            return courseTopicDto;
+        }
+        /// <summary> This method returns List CourseTopicDto, you can choose 1-7 version.</summary>
+        public static List<CourseTopicDto> GetListCourseTopicDto(int vesionOfList)
+        {
+            List<CourseTopicDto> courseTopicsDto = new List<CourseTopicDto>();
+            switch (vesionOfList)
```

Comment:
please try to avoid such switches
it's much better to have several separate methods with programmer-friendly names
it's very hard to understand what are we getting, for example, here: 
var givenTopicsToUpdate = CourseData.GetListCourseTopicDto(6);

it's really cool to move some mocks to this class if they are used in many methods
but if you have a mock for some particular method, it's better to create this mock in the "given" section of the method